### ISSUE TITLE
Input field on main page

### ISSUE THREAD

**Issue by rugk** [small-tech/should-i-pipe-it#7]
The main page should have an input, where I can paste in the URL instead of having to attach it to teh URL.

I.e. a simple redirect.

**Comment by aral** [comment]
Good call.

**Comment by aral** [comment]
Actually, no, given the use case (that it will be linked to from the download page, it doesn’t make sense to have the URL input on the page). The experience starts on some repository/app web page where there will be a link to should-i-pipe.it and people click it to visit the site. Closing.

**Comment by rugk** [comment]
>  the use case (that it will be linked to from the download page

Uh? Nobody told me this is the official™ use case.

The one I had was e.g. this:
1. Found your toot on Mastodon.
2. Click link.
3. Go to main site to enter new URL and check it.
4. Arrg… cannot click there… need to enter it manually…
5. Ctrl+L -> Right button -> paste -> Enter
6. *thinks* : "this should be simpler"
7. *opens issue*

Totally a UX problem, you see?

---

I also don't see why someone should link that on the download page. Then the creator of that link aka the same software could also link to a "fake" page (clone of this) in a phishing-like attack. One would need to check the URL to defeat this.
And, just as with phishing, a good way to solve this is: The user should manually go (type in or bookmark) the official website. (and, in this case, enter the URL there and check it)

BTW, also technically it is not that hard to do. Especially as you already somewhat of have an input field/bar (no joke I tried to click into it when I first saw it!) on the main page.
